### ISSUE TITLE
Record update in Directory Server

### ISSUE THREAD

**Issue by Shourya742** [citadel-tech/coinswap#234]
Currently, the directory server dumps the maker address data only once when its operations are complete. Instead, we want it to store the data at regular intervals during its operation, rather than just at the end.

Possible solution:

We can spawn a new thread that handles this at specific, defined intervals.

**Comment by mojoX911** [comment]
Ack on doing it in a separate thread. Probably for the directory server, we can put it in the server loop itself as a blocking call. 

But the threaded design can be applied to the Maker server too (another issue) to periodically write wallet data to disk.

**Comment by mojoX911** [comment]
As per the discussion. Add the file write operation at handle_client request, everytime it receives a "POST". 

https://github.com/citadel-tech/coinswap/blob/73e17a80ac10d42b97966430d745d1bf90155563/src/market/directory.rs#L252-L255
